docs/creating-service-bindings/binding-options.md

```diff
@@ -6,9 +6,14 @@ sidebar_position: 2
 
```

Comment:
We should have an option that describe/list all the potential options and when a user would be interested by those